server/src/authority/catalog.rs

```diff
@@ -298,6 +303,11 @@ impl Catalog {
         }
     }
 
+    /// Checks whether the `Catalog` contains DNS records for `name`
```

Comment:
Can you ad a comment about the difference between 'contains' and 'find'.

Something to the effect of this must match exactly, where as find will search the name backwards from most specific until least to find a match zone...

Reply:
Done :)

Reply:
perfect, thanks!